navigator_engine/api.py

```diff
@@ -20,37 +20,19 @@ def decide():
                 "url": "<url from estimates dataset json datadict>",
                 "authorization_header": "<optional value to be supplied as the Authorization header tag>"
             },
-            "skipActions": ["<action_id>", "<action_id>"]
+            "skipActions": ["<action_id>", "<action_id>"],
+            "stopAction": "<action_id>"
         }
     ```
     """
-    input_data = json.loads(request.data)
-
-    if not input_data.get('data'):
-        abort(400, "No data specified in request")
-    if not input_data['data'].get('url'):
-        abort(400, "No url to data specified in request")
-
-    graph = load_graph(choose_graph(input_data['data']['url']))
-    data_loader = choose_data_loader(input_data['data']['url'])
-    source_data = input_data['data']
-    skip_requests = input_data.get('skipActions', [])
-    stop_action = input_data.get('actionID')
-
-    engine = DecisionEngine(
-        graph,
-        source_data,
-        data_loader=data_loader,
-        skip_requests=skip_requests,
-        stop=stop_action
-    )
+    engine = create_engine()
     engine.decide()
     del engine.decision['node']
 
-    if stop_action and stop_action != engine.decision['id']:
+    if engine.stop_action and engine.stop_action != engine.decision['id']:
```

Comment:
That's a little misleading. We're including the current decision id in the breadcrumbs. So calling decide with `stopAction = breadcrumbs[-1]['id]` should return a valid decision IMO. What do you think?
I have send exactly this request yesterday and after getting error "The actionID is not in the action path" I thought it's an engine bug.